addons/sale_product_configurator/views/templates.xml

```diff
@@ -177,6 +180,21 @@
                             <t t-call="sale.variants"/>
                         </div>
                     </td>
+                    <td class="text-center td-qty d-none">
+                        <div class="css_quantity input-group oe_website_spinner">
```

Comment:
It is mostly used in the backend tho... I can't even find any css/js code that uses this class. It might be useless. I will dig into it.

Reply:
There are occurences of this css in the themes repository so I think this is probably for website? I'll let you double check.

Reply:
No nothing related to `oe_website_spinner`. Can you look too see if you find anything ?

Reply:
Well as I said it is used in themes, so it should theoretically be only used in website_sale, but let's forget about it, don't want to bother adding an xpath for this.

Reply:
oe_website_spinner style has been deleted (by error imo) by commit 5b65befcd889139152c590490fb8965c9f4b5279 (big diff, so that can happens...)

Purpose was to style this and the inside input and be responsive, centered, ...

In master, this input is broken now:
https://drive.google.com/file/d/1-aaO-cdknNjMVhP8NOoqTzuSl1jHz2eE/view

value disappear, and display is bad

I let you fix design, instead to remove the class ;)

Reply:
This PR will fix the issue: https://github.com/odoo/odoo/pull/35484
The class is no longer needed.